function testErrorFunc(a, func) {         

    //Put your code here

    return x;
}